src/from_substrait.cpp

```diff
@@ -778,6 +804,19 @@ shared_ptr<Relation> SubstraitToDuckDB::TransformRootOp(const substrait::RelRoot
 		}
 	}
 
+	if (sop.input().rel_type_case() == substrait::Rel::RelTypeCase::kWrite) {
+		auto write = sop.input().write();
+		switch (write.op()) {
+		case substrait::WriteRel::WriteOp::WriteRel_WriteOp_WRITE_OP_CTAS: {
+			const auto create_table = static_cast<CreateTableRelation *>(child.get());
+			auto proj = make_shared_ptr<ProjectionRelation>(create_table->child, std::move(expressions), aliases);
+			return proj->CreateRel(create_table->schema_name, create_table->table_name);
+		}
+		default:
+			return child;
```

Comment:
returning child here seems weird. what's the reasoning?

Reply:
It'd be useful to document this branch as well with the answer.  I'm guessing we're treating this as a no-op for all non-write operations.

Reply:
I don't understand why would we wouldn't throw here. It sounds like you're saying that code above is going to throw so we just return something random here. And what's wrong with adding a projection relation?

Reply:
Let's merge this change and open an issue to follow-up on better clarity around why we have these weird specialized cases outside of individual transform ops.